config/locales/en.yml

```diff
@@ -54,16 +54,26 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the_work: "Edit the work"
+      edit_the_series: "Edit the series"
+      part1_work: "You have been listed as a coauthor on the following work:"
+      part1_series: "You have been listed as a coauthor on the following series:"
+      html:
+        part2_work: "%{edit} to remove yourself as author if you've been added in error or don't want to be listed as a creator."
```

Comment:
As much as it pains me, the "creator" at the end of all these should stay "author" unless we're going to update the whole thing to use "creator" and "co-creator". (And if we do, pretty please let's make the capitalization in the subject "Co-creator notification" -- we use a lot of sentence case subject lines and they look _so_ much better.)